cmake/ament_doxygen_generate.cmake

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Going out on a bit of a limb here, but I think we should add the license header for Apache 2.0 with a copyright holder of Toyota Research Institute here.  @stonier can you confirm?

Reply:
Copyright Toyota Research Institute / License Apache 2.0 is fine by me.